content/en/quality_gates/explorer/facets.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Quality Gates or Rule Execution Facets
+kind: documentation
+description: Learn about facets in the Quality Gates Explorer.
+further_reading:
+- link: 'quality_gates/explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the Quality Gates Explorer'
+- link: 'quality_gates/search/'
+  tag: 'Documentation'
+  text: 'Learn how to search your rules and executions'
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Quality Gates is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" btn_hidden="true" >}}
+Quality Gates is in public beta.
+{{< /callout >}}
+
+## Overview
+
+Facets are user-defined tags and attributes from your rules or executions. They are useful for both [qualitative](#qualitative-facets) and [quantitative](#quantitative-measures) data analysis. Facets allow you to manipulate your rules or executions in search queries that appear on [dashboards][2] and in [notebooks][3].
+
+[Creating facets](#creating-facets) is **not required** for [searching quality gates][5]. Autocomplete capabilities use existing facets, but also any input that matches incoming quality gates applies.
+
+The [Quality Gates Explorer][4] includes out-of-the-box facets such as `Test Status`, `Test Service`, `CI Status`, and `CI Provider`. You can use facets in the Quality Gates Explorer to [search for and filter quality gates][5].
```

Comment:
These out of the box facets are leftover from Test, can we update with Quality Gate facets

Reply:
```suggestion
The [Quality Gates Explorer][4] includes out-of-the-box facets such as `Status` and `Gate ID`. You can use facets in the Quality Gates Explorer to [search for and filter quality gates][5].
```